src/modules/score.js

```diff
@@ -1,53 +1,44 @@
 class Score {
   constructor() {
     this.showData = document.querySelector('.dom-show');
-    this.collection = JSON.parse(localStorage.getItem('scoreData')) || [
-      {
-        id: 1,
-        name: 'Muneeb',
-        score: 100,
-      },
-      {
-        id: 2,
-        name: 'Talha',
-        score: 100,
-      },
-      {
-        id: 3,
-        name: 'Waqar',
-        score: 100,
-      },
-      {
-        id: 4,
-        name: 'Faisal',
-        score: 100,
-      },
-      {
-        id: 5,
-        name: 'Ali',
-        score: 100,
-      },
-    ];
+    this.collection = [];
+    this.link = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/wVpJdekyVeWw4P4hGrPS/scores/';
   }
 
+  fetchData = () => {
+    fetch(this.link)
+      .then((response) => response.json())
+      .then((data) => {
+        this.collection = data.result;
+        return this.renderData();
+      })
+      .catch((error) => {
+        console.error('Error:', error);
+      });
+  };
```

Comment:
According to the project requirements, you are supposed to use the `async` and `await` javascript features to consume the API. Please update your functions for fetching and posting data to the API to use async and await keywords🙏